Core/Lib/Models/QueryModel.cs

```diff
@@ -18,9 +18,12 @@ public class QueryModel
     public bool NoLimit { get; set; } = false;
 
     // Filter by fields:
+    [Obsolete("Tag is deprecated and might be remove in a future version of the framework. Please use the Tags instead.")]
```

Comment:
For the time being, do `Tag` **and** `Tags` work?

Reply:
Yes, both Tag and Tags works as parameters on QueryModel (both are strings, with the ideea that in the Tag you should use one tag and in the Tags you can send multiple tags as a string separated by comman). Tag is marked as deprecated/obsolete.